cms/models.py

```diff
@@ -38,3 +39,33 @@ def get_context(self, request, *args, **kwargs):
         context = super(BootcampPage, self).get_context(request)
         context["title"] = self.title
         return context
+
+
+class ResourcePage(Page):
+    """
+    Basic resource page for all resource page.
+    """
+    template = "../../bootcamp/templates/resource_template.html"
```

Comment:
This path seems to be incorrect

Reply:
loading fine at my end. further elaborate ?

Reply:
updated: I think, something has been updated in settings.py file after rebase. anyway, I will do the corresponding fix.

Reply:
<img width="807" alt="Screen Shot 2020-04-30 at 10 11 24 PM" src="https://user-images.githubusercontent.com/7574259/80776139-98b5da80-8b2f-11ea-8cf0-89bb11134289.png">

Reply:
Yes @annagav you are right. @asadiqbal08 gavin has renamed the base app from `bootcamp` to `main` so all your bootcamp app changes should be in `main` app. That's the issue here.